test/lib/core_extensions_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class CoreExtensionsTest <  ActiveSupport::TestCase
+  context 'string' do
+    test '#to_gb' do
+      value = "1024 MB"
+      to_gb_value = value.to_gb
+      assert_equal(1.0, to_gb_value, "Converted value should be 1.0")
+      assert(to_gb_value.is_a?(Float), "Converted value shoud be a float")
+    end
+
+    test '#to_utf8' do
+      value = Rails.public_path
+      to_utf8_value = value.to_utf8
+      assert_equal(Encoding::UTF_8, to_utf8_value.encoding)
```

Comment:
Good question,
In my local setup `Rails.public_path.encoding` returns `<Encoding:US-ASCII>` so I test that it was transformed to UTF-8
I can add an assertion which creates a string with `ASCII-8BIT` and test if it transform into utf-8.
Sounds good?